nixos/modules/system/activation/activation-script.nix

```diff
@@ -154,9 +154,15 @@ in
 
     system.activationScripts.tmpfs =
       ''
-        ${pkgs.utillinux}/bin/mount -o "remount,size=${config.boot.devSize}" none /dev
```

Comment:
Regardless, I still think that resetting options for all special file systems here is a good idea because with systemd update or otherwise may come new mount options that we can set without rebooting.